### ISSUE TITLE
UX improvement: shortcut for New node

### ISSUE THREAD

**Issue by Poulpator** [nodebox/nodebox#443]
In a node based application use the mouse  is a condition sine qua none  to use the software  but we also often add a new node,
Here the problem, as today the shortcut to add a new node is ctrl+alt+n which make you use your two hands therefore you cannot use the mouse in the same time.
To speed up the workflow, I recommand to move the shortcut to a key more on the left of the keyboard.
I personnaly like maj+A which stand for add, i have extensivly use this shortcut in blender3D

**Comment by Shrinks99** [comment]
I would second this comment though I wouldn't go with shift+a, Blender's default shortcuts are not exactly "industry standard" as they'd be the first to admit.

I would instead suggest the tab key, the default shortcut for adding new nodes in Nuke, Natron, Maya, Blender's "Industry" keymap, and Houdini... among others.

ctrl+shift+A is a really bad choice for this shortcut.  Adding nodes to the graph is an incredibly common task and should be made as friction-less as possible.  I would highly recommend condensing it to the tab key, this will result in serious speed improvements and will make users with experience in any other node-based software feel more at home in Nodebox.

**Comment by rachellewillemsma** [comment]
Agreed, having the tab key as a new node shortcut would be drastically more efficient. I wasn't even aware there was a shortcut until now. Adding nodes is pretty much the primary thing a user does in this program, so would make sense to make the shortcut as short as possible.

**Comment by fdb** [comment]
Currently, you can **double-click** to place a new node where you want it to appear. This solves the problem of NodeBox having to figure out *where* to place the new node...

**Comment by Shrinks99** [comment]
Nodebox already does a reasonably good job of placing nodes in the graph when users utilize the shortcut, type to find the node they want, and hit enter.

Nodebox has integrated its graphical discovery system and text interface for adding new nodes to the graph within the same window.  This is all well and good, no complaints here.  The problem this issue is trying to address is the non-standard and slow shortcut for opening the node add window.

**Comment by rachellewillemsma** [comment]
Yes, the problem is with the need to use the mouse every time a new node is added. It seems using Ctrl+Shift+N simply places the node in the centre of the current view. I see no reason why a shorter keyboard shortcut could not do the same.
